### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by mchompalova**
# Description

## What

Add runbook for the service.

## Why

https://financialtimes.atlassian.net/browse/UPPSF-1181

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by bboykov — DISMISSED** [review]

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Known About By
+
+- dimitar.terziev
+- elitsa.pavlova
+- kalin.arsov
+- hristo.georgiev
+- tsvetan.dimitrov
+- elina.kaneva
+- robert.marinov
```
Comment (ekaneva) [inline runbooks/runbook.md:37]: May be add one more Content team member, e.g. Georgi Ivanov :)

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+- upp-neo4j-cluster
```
Comment (ekaneva) [inline runbooks/runbook.md]: Neo4j is not a dependancy here, but both articles mappers and Methode API are:
- <https://biz-ops.in.ft.com/System/up-mam>
- <https://biz-ops.in.ft.com/System/up-maicm>
- <https://biz-ops.in.ft.com/System/up-mapi>

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
+# UPP - Content Preview
+
+Content Preview accesses unpublished content and previews it before publishing.
```
Comment (ekaneva) [inline runbooks/runbook.md]: This information is not completely true. It is not a preliminary check before publishing. The services provides an option to preview content (mainly published from Methode) closely to what it is going to look like on ft.com. Please edit it a bit.
You can shorten and use the first sentence from the Architecture part/ the README description.

**Review by ekaneva — CHANGES_REQUESTED** [review]

File: runbooks/runbook.md (hunk 2 above)
Comment (mchompalova) [reply runbooks/runbook.md]: Copy-paste mistake, I'll fix it.

**Review by ekaneva — APPROVED** [review]
Looks good now.

**Review by bboykov — APPROVED** [review]

**Pull request merged**